app/src/main/java/com/kickstarter/ui/views/KSWebView.kt

```diff
@@ -127,7 +125,7 @@ class KSWebView : FrameLayout, KSWebViewClient.Delegate {
     }
 
     fun evaluateJavascript(javascript: String?, resultCallback: ValueCallback<String>?) {
-        internal_web_view.evaluateJavascript(javascript, resultCallback)
+        javascript?.let { internal_web_view.evaluateJavascript(it, resultCallback) }
```

Comment:
Is this change related to adding these new properties?

Reply:
not really, I just saw it and fixed it